README.md

```diff
@@ -2,6 +2,11 @@
 
 Kubernetes Operator which implements the backend for [Lieutenant API](https://github.com/projectsyn/lieutenant-api).
 
+The operator keeps inventory about all the tenants and clusters in a SYN managed k8s cluster.
+
+It also handles the management of some requirements like Git Repositories and secret management. It can automatically populate Git repositories with skeleton files when a new cluster is added. It will also generate a Steward secret.
```

Comment:
```suggestion
It also handles the management of some requirements like Git repositories and secret management: It can automatically populate Git repositories with template files when a new cluster is added. It will also generate a token to be used by Steward.
```